ember-load-initializers.js

```diff
@@ -8,7 +8,7 @@ define("ember/load-initializers",
       'default': function(app, prefix) {
         var regex = new RegExp('^' + prefix + '\/((?:instance-)?initializers)\/');
 
-        Ember.keys(requirejs._eak_seen).map(function (moduleName) {
+        Object.keys(requirejs._eak_seen).map(function (moduleName) {
```

Comment:
Lets fall back if not present (this will keep us in line with whatever Ember version is being used).